From c996107987454cd5c52d57bd18b8b0a08ab488b5 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Sun, 7 Oct 2018 10:51:03 +0800
Subject: [PATCH] clk: rockchip: rk1808: mark some clk as critical clocks

mark "pclk_top_pre","pclk_ddr_grf","aclk_gic" as critical clocks

Change-Id: I4daf50c36e899edcff07778cbb98571c2ed2b042
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk1808.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/clk/rockchip/clk-rk1808.c b/drivers/clk/rockchip/clk-rk1808.c
index 5c13de1869ea..ecf95776696b 100644
--- a/drivers/clk/rockchip/clk-rk1808.c
+++ b/drivers/clk/rockchip/clk-rk1808.c
@@ -1151,6 +1151,9 @@ static const char *const rk1808_critical_clocks[] __initconst = {
 	"msclk_bus_niu",
 	"lsclk_bus_niu",
 	"pclk_pmu_niu",
+	"pclk_top_pre",
+	"pclk_ddr_grf",
+	"aclk_gic",
 };
 
 static void __iomem *rk1808_cru_base;
-- 
2.35.3

